src/ontology/config/property-map.sssom.tsv

```diff
@@ -7,3 +7,4 @@ http://www.ebi.ac.uk/efo/reason_for_obsolescence	rdfs:comment
 http://www.ebi.ac.uk/efo/alternative_term	oboInOwl:hasExactSynonym
 http://purl.obolibrary.org/obo/ECO_0000218	oboInOwl:source
 http://purl.obolibrary.org/obo/IAO_0000118	oboInOwl:hasExactSynonym
+http://purl.obolibrary.org/obo/OMO_0003000	http://purl.obolibrary.org/obo/mondo#ABBREVIATION
```

Comment:
Remember to update this branch with updates from https://github.com/monarch-initiative/mondo-ingest/pull/663 so this line will be removed.

Reply:
Done!